source/tests/system/nirfmxnr_session_tests.cpp

```diff
@@ -111,7 +111,6 @@ TEST_F(NiRFmxNRSessionTest, InitializedSession_CloseSession_ClosesDriverSession)
 
 TEST_F(NiRFmxNRSessionTest, TwoInitializedSessionsOnSameDevice_CloseSessions_ClosesDriverSessions)
 {
-  GTEST_SKIP() << "Initializing two sessions on same device is currently broken. Planning to fix with AB#1792683";
```

Comment:
This test used to fail on the last line where it expected the second close call's `close_response_two` to be good. It was getting an invalid session error though.

The flow through the test previously was
```c++
Init session one
    Session_Resource mappings
        map [{1, 0x01}]
        reverse_map [{0x01, 1}]
    Session repository mappings
        [{Id: 1, {1, }}]
        [Name: "SessionOne", {1, ...}]
Init session two
    Session_Resource mappings
        map [{1, 0x01}, {2, 0x01}]
        reverse_map [{0x01, 2}] // Overwrites pair with key 0x01 with new session id: 2
    Session repository mappings
        [{Id: 1, {1, }}, {Id: 2, {2, ...}}]
        [{Name: "SessionOne", {1, ...}}, {Name: "SessionTwo", {2, ...}}]
Close session one
    Session_Resource mappings
        map [{1, 0x01}] // Here, it gets session id of handle (0x01) which returns 2. So 2 key is removed.
        reverse_map [] // Empty, there was only one key/value pair in here.
    Session repository mappings
        [{Id: 1, {1, }}}]
        [{Name: "SessionOne", {1, ...}}] // Removes wrong one since it gets wrong one through Handle => session map
Close session two
     // Can't find session with id 2, it was removed.
     // So it goes through with a 0 session id and 0 for the session handle causing failure.
```